src/widgets/modals/ByProgramModal.js

```diff
@@ -279,8 +296,7 @@ const localStyles = StyleSheet.create({
   okButton: {
     ...globalStyles.button,
     backgroundColor: SUSSOL_ORANGE,
-    alignSelf: 'center',
-    marginTop: 60,
+    marginTop: 'auto',
```

Comment:
I learnt this cool trick and I wanted to use it :( (Flex auto margins expand to fill the space)